packages/grafana-data/src/types/transformations.ts

```diff
@@ -42,6 +46,10 @@ export interface DataTransformerConfig<TOptions = any> {
    * Options to be passed to the transformer
    */
   options: TOptions;
+  /**
+   * Function to apply template variable substitution to the DataTransformerConfig
+   */
+  replace?: (target?: string, scopedVars?: ScopedVars, format?: string | Function) => string;
```

Comment:
this feels pretty bad to have it on the persisted option model like this

Reply:
agree -- accidents like this are what we hope to avoid with `@grafana/schema`... but so many things 😬